src/node/util.ts

```diff
@@ -388,6 +391,14 @@ export function styleText(
   return skipColorize ? text : `${left}${text}${right}`;
 }
 
+export function _errnoException(): void {
+  throw new ERR_METHOD_NOT_IMPLEMENTED('_errnoException');
+}
```

Comment:
Are we planning to implement this at any point? If we are, then a TODO comment along those lines would be helpful. If we aren't, then a comment indicating that too would be helpful